src/extra/geotiff.jl

```diff
@@ -2,16 +2,28 @@
 # Licensed under the MIT License. See LICENSE in the project root.
 # ------------------------------------------------------------------
 
+struct GDALTransform{T} <: CoordinateTransform
+  gt::NTuple{6,T}
+end
+
+GDALTransform(gt::AbstractVector) = GDALTransform(ntuple(i -> gt[i], 6))
+
+function applycoord(transform::GDALTransform, vec::Vec{2})
+  x, y = vec
+  gt = transform.gt
+  xnew = gt[1] + x * gt[2] + y * gt[3]
+  ynew = gt[4] + x * gt[5] + y * gt[6]
+  Vec(xnew, ynew)
+end
```

Comment:
@ErickChacon can you confirm that the GDAL transform operates on the original x and y of the cartesian grid? The docstring of the transform only mentions `pixel` and `line` as the input to this transform constants. We are assuming they refer to the original array coordinates.

Reply:
![image](https://github.com/JuliaEarth/GeoIO.jl/assets/3345261/b543d178-aa93-42f0-bfb4-9a60a56892a6)

Reply:
@juliohm, sorry for the late reply. Yes, this is the way the affine transformation is done. We probably have also to modify the function to plot TransformedGrids? With the previous code, I could quickly visualize the loaded tiff file. With the current code it creates files that my computer does not show anymore, probably because it is not using the heatmap function.